package.json

```diff
@@ -15,7 +15,8 @@
   "scripts": {
     "build": "cross-env NODE_ENV=production gulp build",
     "build:changelog": "github_changelog_generator --no-issues --no-unreleased --release-branch master --since-tag $(git describe --abbrev=0 --tags $(git rev-parse HEAD~300))",
-    "build:docs": "gulp --series build:docs",
+    "build:docs": "cross-env NODE_ENV=production gulp build:docs",
+    "build:docs:staging": "cross-env STAGING=true yarn build:docs && yarn serve docs/dist",
```

Comment:
This for testing production build of docs locally

Reply:
NODE_ENV=test?

Reply:
The main idea that build will be production, so you will have the same result as will be on `react.semantic-ui.com`, i.e. manual predeployment check.

The current production build cannot be tested locally because it has hardcoded links to `react.semantic-ui.com`.

So we can't use there `NODE_ENV` variable, however we can remove this change at all. I thought that it would be useful for you.